### ISSUE TITLE
Minimum time interval between successive requests to same endpoint

### ISSUE THREAD

**Issue by sandeepbol** [Moya/Moya#1677]
Is there a concept of minimum time interval between successive requests to same endpoint in Moya?

**Comment by BasThomas** [comment]
What is the use case? I feel like this is a responsibility of the client.

**Comment by sandeepbol** [comment]
Suppose in viewwillAppear(), we fetch data from endpoint. User can return to viewcontroller by pressing a tab button (say). If I keep hitting tab button then I keep hitting the endpoint with mostly no new data arriving.

**Comment by BasThomas** [comment]
That does feel like a client side thing. 

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by sandeepbol** [comment]
Moya (Alamofire) is felicitating the API requests here right? So either Moya or Alamofire should have this feature right?

**Comment by SD10** [comment]
@sandeepbol Yeah I'm a little confused as to whether the behavior you want here is something like tracking in flights or throttling requests, but this feature is out of scope for Moya. It's too client specific
